common/oatbox/extension/Manifest.php

```diff
@@ -249,6 +255,16 @@ public function getConstants(): array
             ? $this->manifest['constants'] : [];
     }
 
+    /**
+     * Get an array of roles to be handled by the extension (a role hierarchy).
+     * @return array
+     */
+    public function getIncludedRoles(): array
```

Comment:
Maybe return type can be avoided in the docs since it is explicitly declared? 

```suggestion
    /**
     * Get an array of roles to be handled by the extension (a role hierarchy).
     */
    public function getIncludedRoles(): array
```

Reply:
done, my mistake for copy and pasting :facepalm: thanks! :D